tracer/src/Datadog.Trace.Bundle/Datadog.Trace.Bundle.targets

```diff
@@ -0,0 +1,77 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="RemoveDatadogNuGetFiles" AfterTargets="Publish" Condition="'$(RemoveUnusedDatadogFiles)' == 'true'">
+    <PropertyGroup>
+      <!-- Root where Datadog bundle files are published -->
+      <BundleRoot>$(PublishDir)datadog\</BundleRoot>
+
+      <!-- Helper properties -->
+      <IsNetFramework>$(TargetFramework.StartsWith('net4'))</IsNetFramework>
+
+      <!-- Map TargetFramework to required folder -->
+      <EffectiveTargetFramework Condition="$(IsNetFramework)">net461</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netstandard2.0'">netstandard2.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp2.0'">netstandard2.0</EffectiveTargetFramework>
+      <EffectiveTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp2.1'">netstandard2.0</EffectiveTargetFramework>
```

Comment:
For completeness, can we include `netcoreapp2.2`?

Reply:
Done.